### ISSUE TITLE
jei does not always show nitric and sulfuric acid recipes 

### ISSUE THREAD

**Issue by kiwifrogg** [GlobbyPotato/Rockhounding_Chemistry#29]
I did mention this in another issue, but it might need its own Sulfuric Acid needs  sulfur bearing compound ,sodium hydroxide needs chloride bearing compound . But this does not always show in the nei and when you look through the recipies it does not show there either. Hower sometimes when its just the core mod and RHC it comes up just fine.  We do have mek and Galactic installed they do mess with some of the fluids recipes so that could be the base cause.

**Comment by GlobbyPotato** [comment]
Sulfuric acid and sodium hydroxide should be fixed now, it seems to be a missing null check that makes them invalid in jei. About the nitric acid I can't see problems yet, I always see the recipe by pressing R or U in jei even with both mek and GC installed.

I noticed mekanism seems to use the same oxygen registry naming, but I was able to produce both separately in the GAN plant and Electrolytic Separator with no conflicts. Galacticraft should use also some other fluids identically named but never used the mod so I can't figure out how they are handled.
